azurelinuxagent/common/conf.py

```diff
@@ -127,7 +128,8 @@ def load_conf_from_file(conf_file_path, conf=__conf__):
     "ResourceDisk.MountPoint" : "/mnt/resource",
     "ResourceDisk.MountOptions" : None,
     "ResourceDisk.Filesystem" : "ext3",
-    "AutoUpdate.GAFamily" : "Prod"
+    "AutoUpdate.GAFamily" : "Prod",
+    "Provisioning.Endpoint" : "168.63.129.16"
```

Comment:
Our contract is that we discover this via DHCP.  The value is always the same, but that's an implementation detail. Azure is free to change this value, and may. I believe this value is different in Azure Stack.